sdk/eventhub/Azure.Messaging.EventHubs/tests/Producer/EventDataBatchTests.cs

```diff
@@ -57,10 +94,11 @@ public void ConstructorUpdatesState()
         ///   class.
         /// </summary>
         ///
+        [Test]
```

Comment:
Wow.... what the heck did I do?   Leave the test decorator off of everything?   I guess that's one way to make sure everything passes.